cli/tauri-cli/src/bundle/templates/bundle_dmg

```diff
@@ -128,6 +128,9 @@ hdiutil detach ${device}
 
 hdiutil convert "${DMG_TEMP_NAME}" -format UDBZ -imagekey zlib-level=9 -o "${MACOS_APP_NAME}"
 
+wget -nc https://github.com/sindresorhus/create-dmg/raw/c9b232da8fc2d7068a85f227d3a85bd8210b5bcd/seticon
+chmod +x seticon
```

Comment:
this is already available. its shipped in cli/tauri-cli/src/bundle/templates/seticon
and is wired up to be consumed by cli/tauri-cli/src/bundle/templates/bundle_dmg

Reply:
Ah, whoops. I missed that